### PR TITLE
some updates

### PR DESCRIPTION
**Pull request by bourgeoa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -280,17 +280,15 @@ Make sure you have the needed environment: [nvm for SolidOS](https://github.com/
...
+npm run prepare # to install the requested node versions
+# ... and maybe missing versions if build errors tell you so ...
+# nvm install xxx # version missing
 npm run setup
 ```
 
 Run the above lines in a terminal of your choice to setup your SolidOS project folder. By default, some dependent repos are also set up for you:
 
 - [rdflib.js](https://github.com/linkeddata/rdflib.js): Javascript RDF library for browsers and Node.js
-- [solid-logic](https://github.com/solid/solid-logic): core business logic of SolidOS
```
Comment (theRealImy) [inline README.md:293]: this actually gets me to the idea we should have solid-logic also as a default repo pulled in. How is this set up? Can we do a follow up git issue about it, does it make sense?

**Review by theRealImy — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (bourgeoa) [reply README.md:293]: This is in scripts/reset

Comment (theRealImy) [reply README.md:293]: I suppose we should yes... then it is clear which subrepos we have.

Comment (theRealImy) [reply README.md:293]: I can do it and also change the readme. So you can merge this one.

Comment (theRealImy) [reply README.md:293]: see: https://github.com/solid/solidos/issues/131

**Review by theRealImy — APPROVED** [review]

**Pull request merged**